flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -280,6 +316,55 @@
 	 */
 	void sqlUpdate(String stmt, QueryConfig config);
 
+	/**
+	 * Gets the current default catalog name of the current session.
+	 *
+	 * @return the current default catalog that is used for path resolution
+	 * @see TableEnvironment#setCurrentCatalog(String)
+	 */
+	String getCurrentCatalogName();
```

Comment:
Nit: I wonder if we just name it getCurrentCatalog(), which returns the name of the current catalog. (Same for get current database name. I understand that the given name is more explicit, but getCurrentCatalog() is a mirror of the set method below and is more consistent with  catalog APIs in which, every get (table, database) is named getXxx() instead of getXxxName().